packages/playwright/src/index.ts

```diff
@@ -632,6 +634,7 @@ class ArtifactsRecorder {
 
     this._pageSnapshotRecorder = new SnapshotRecorder(this, pageSnapshot, 'pageSnapshot', 'text/yaml', '.aria.yml', async (page, path) => {
       const ariaSnapshot = await page.locator('body').ariaSnapshot({ timeout: 5000 });
+      this._pageSnapshot ??= ariaSnapshot;
       await fs.promises.writeFile(path, ariaSnapshot);
```

Comment:
Do we need to store it in a file with the present design?

Reply:
nope, but `SnapshotRecorder` demands that for how its temporary snapshots work. I replaced this with a simpler impl that doesn't store into file